examples/farmer.js

```diff
@@ -1,12 +1,12 @@
-const Vec3 = require('vec3')
-const mineflayer = require('mineflayer')
+import Vec3 from 'vec3'
```

Comment:
And now vec3 without destruction, @Karang which version is valid at the moment?

Reply:
It should be `{ Vec3 }`